src/clawback_escrow/src/test/clawback.test.ts

```diff
@@ -0,0 +1,171 @@
+import { TokenContractArtifact, TokenContract } from '../../../artifacts/Token.js';
+import { EscrowContractArtifact, EscrowContract, PrivacyKeys } from '../../../artifacts/Escrow.js';
+import { ClawbackEscrowContractArtifact, ClawbackEscrowContract } from '../../../artifacts/ClawbackEscrow.js';
+import {
+  AccountWallet,
+  createLogger,
+  Fr,
+  PXE,
+  Logger,
+  AztecAddress,
+  AccountWalletWithSecretKey,
+  Wallet,
+  Note,
+  UniqueNote,
+  ContractDeployer,
+} from '@aztec/aztec.js';
+import { createAccount } from '@aztec/accounts/testing';
+import { computePartialAddress, deriveKeys, getContractInstanceFromDeployParams } from '@aztec/circuits.js';
+import { createPXE, expectTokenBalances, logger, wad } from '../../../ts/test/utils.js';
+import { deployToken } from '../../../ts/test/token.test.js';
+
+async function deployEscrow(pxes: PXE[], deployerWallet: Wallet, owner: AztecAddress): Promise<EscrowContract> {
+  const escrowSecretKey = Fr.random();
+  const escrowPublicKeys = (await deriveKeys(escrowSecretKey)).publicKeys;
+  const escrowDeployment = EscrowContract.deployWithPublicKeys(escrowPublicKeys, deployerWallet, owner);
+  const escrowInstance = await escrowDeployment.getInstance();
+  await Promise.all(
+    pxes.map(async (pxe) => pxe.registerAccount(escrowSecretKey, await computePartialAddress(escrowInstance))),
+  );
```

Comment:
bob shouldn't be learning about the escrow keys yet, only Alice registers the account here